### ISSUE TITLE
Offline rendering mode (for oversampling)

### ISSUE THREAD

**Issue by dyross** [vvvar/PeakEater#74]
**Is your feature request related to a problem? Please describe.**
It would be nice to allow higher level of oversampling at render time than during mixing.

**Describe the solution you'd like**
Maybe plugins, like KClip3, support this. Could be just another knob on the UI.

**Comment by joetink** [comment]
Agreed.  Offline oversampling would be crucial for projects that are already using a lot of cpu and can't afford to run multiple instances of this at 16x.

**Comment by justinwelgraven** [comment]
This would be awesome.

**Comment by joetink** [comment]
Anyone know if the developer is still active?  Offline oversampling would be an amazing addition to this plugin that would quite honestly put the plugin on the level of K-Clip, V-Clip, Standard Clip, etc.
